src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
@janvorli Yes, it seems that the latter encoding corresponds to the current x86 implementation, but I couldn't understand why the former (direct) encoding works..